src/objects.rs

```diff
@@ -345,15 +351,13 @@ impl Id<StopArea> for StopPoint {
 }
 impl_codes!(StopPoint);
 
-pub type Date = String;
```

Comment:
Better to keep this Date type and use it everywhere, it's simpler if we want to change.

Thus, something like `pub type Date = chrono::NaiveDate;`

Reply:
fixed